src/tests/make_test_list.py

```diff
@@ -33,7 +93,13 @@ def collect_tests(base_dir):
 
 
 if __name__ == "__main__":
+    old_error_flags = collect_old_error_flags()
     tests = collect_tests(TEST_DIR)
     with open("python_tests.txt", "w") as fp:
+        fp.write(explanation)
         for test in tests:
-            fp.write(test + "\n")
+            error_flags = " ".join(old_error_flags.get(test, []))
+            line = test
+            if error_flags:
+                line += "\t" + error_flags
```

Comment:
Could we use 4 space instead? Or some number of spaces anyway?
```
```suggestion
                line += "    " + error_flags
```

Reply:
Before, the file had spaces to ensure vertical alignment and putting tabs approximates visual alignment without having to count spaces. But that isn't all that important of course.